zhaquirks/philips/soc001.py

```diff
@@ -0,0 +1,45 @@
+"""Signify SOC001 device."""
+
+from zigpy import types
+from zigpy.quirks import CustomCluster
+from zigpy.quirks.v2 import BinarySensorDeviceClass, ClusterType, QuirkBuilder
+
+
+class PhilipsContactCluster(CustomCluster):
+    """Philips manufacturer specific cluster for contact sensor."""
+
+    cluster_id = 64518  # 0xfc06
+    name = "Philips contact cluster"
+    ep_attribute = "philips_contact_cluster"
+    attributes = {
+        0x0100: ("contact", types.enum8, True),
+        0x0101: ("last_contact_change", types.uint32_t, True),
+        0x0102: ("tamper", types.enum8, True),
+        0x0103: ("last_tamper_change", types.uint32_t, True),
+    }
+
+
+(
+    #  <SimpleDescriptor endpoint=2 profile=260 device_type=1026
+    #  device_version=0
+    #  input_clusters=[0, 1, 3, 64518]
+    #  output_clusters=[0, 3, 6, 25]>
+    QuirkBuilder("Signify Netherlands B.V.", "SOC001")
+    .removes(cluster_id=6, endpoint_id=2, cluster_type=ClusterType.Client)
```

Comment:
Just noting:
Since the `OnOff` cluster is removed here, this would be a breaking change (and also makes it impossible to configure binding via the UI).

Reply:
Changed in : https://github.com/zigpy/zha-device-handlers/commit/e101ca095ae4a7d6e21f34bd49d587de7c575ea2